src/translib/transformer/xlate_from_db.go

```diff
@@ -491,7 +491,8 @@ func dbDataFromTblXfmrGet(tbl string, inParams XfmrParams, dbDataMap *map[db.DBN
 func yangListDataFill(dbs [db.MaxDB]*db.DB, ygRoot *ygot.GoStruct, uri string, requestUri string, xpath string, dbDataMap *map[db.DBNum]map[string]map[string]db.Value, resultMap map[string]interface{}, tbl string, tblKey string, cdb db.DBNum, validate bool, txCache interface{}, isFirstCall bool) error {
 	var tblList []string
 
-	if tbl == "" && xYangSpecMap[xpath].xfmrTbl != nil {
+	_, ok := xYangSpecMap[xpath]
+	if tbl == "" && ok && xYangSpecMap[xpath].xfmrTbl != nil {
```

Comment:
can we just check ok once, instead of checking multiple ifs below?

Reply:
Done